src/main/resources/assets/ironchest/lang/fi_FI.lang

```diff
@@ -0,0 +1,177 @@
+################
+#    Chests    #
+################
+
+tile.ironchest.chest.iron.name=Rauta arkku
```

Comment:
These should be
Rautainen
Kultainen
Timanttinen
Kuparinen
Hopea-arkku
Multa-arkku
The rest seems ok. I'm from Finland and am Finnish so those are my tips.

Reply:
Kyl mie kin on suomesta mut ei vaan kina tuu sita kultaista pistetta loydettya, ku on kysynyt kavereilta niin mielet jakautuivat kahtia. Niin paattin jatta nain